### DISCUSSION TITLE
GraphicsMagick support

### DISCUSSION THREAD

**Discussion by ammannbe** [craftcms/cms#15576] [Ideas]
Hi there

As I saw in [this old post](https://craftcms.stackexchange.com/questions/29784/can-i-use-graphicsmagick-with-craft-cms), graphicsmagick is not supported by Craft CMS and that's probably still the case.
The image transformation on our hosting is very slow with imagick but they would support GraphicsMagick as a fast alternative.

Would it be possible to implement this as an additional image driver?

**Comment by angrybrad** [comment]
It would take some non-trivial effort, and to be honest, there haven't been many requests for it.

If GD isn't an option for you (which PHP supports natively), I'd probably focus on my Imagick is performing slowly for you on that host. Are they running an older version? Are they limited the resources it can access via a policy.xml file? etc.
